addons/crm_livechat/models/crm_lead.py

```diff
@@ -0,0 +1,15 @@
+import json
+from ast import literal_eval
+
+from odoo import fields, models
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    channel_id = fields.Many2one("discuss.channel", "Live chat from which the lead was created")
```

Comment:
Missing 'btree_not_null` index.
Ref to https://github.com/odoo/enterprise/pull/76279#discussion_r1917044534 for more info